cmd/erasure-object.go

```diff
@@ -692,12 +687,14 @@ func (er erasureObjects) getObjectFileInfo(ctx context.Context, bucket, object s
 			missingBlocks++
 			continue
 		}
-		if metaArr[i].IsValid() && metaArr[i].ModTime.Equal(fi.ModTime) {
-			continue
+		if etag == "" {
```

Comment:
Isn't this a little bit downgrade of a feature, before we can detect disks without outdated xl.meta, but now this only works when etag == 0 (no common etag?)

Reply:
This will be empty always since modTime is not equal to zero values unless quorum can't be achieved